### ISSUE TITLE
Add option for local date

### ISSUE THREAD

**Issue by thecesrom** [orhun/git-cliff#70]
**Is your feature request related to a problem? Please describe.**
Yesterday, while it still was 2022-03-15 18:35 in my time zone, `git-cliff` set the date to `2022-03-16`. See [here](https://github.com/ignition-api/7.9/blob/main/CHANGELOG.md#7919---2022-03-16).

While I do believe that someone in a time zone where it already was 2022-03-16 wouldn't see anything weird, I was wondering if it would be possible to let the maintainer choose the date they want.

**Describe the solution you'd like**
Add option to `cliff.toml`.

**Describe alternatives you've considered**
Leave as-is.

**Additional context**
None.

**Comment by orhun** [comment]
Hello!

It is possible to set a specific timezone for the dates in changelog using the [`date`](https://tera.netlify.app/docs/#date) Tera filter. So for example, if you have this:

```
{{ timestamp | date(format="%Y-%m-%d") }}
```

You can supply a timezone like this:

```
{{ timestamp | date(timezone="America/New_York", format="%Y-%m-%d") }}
```

I hope this helps 🐻

**Comment by thecesrom** [comment]
This sure helps!

Thanks you, and apologies for not reading the docs.

**Comment by thecesrom** [comment]
I have tried the suggested approach, but in my case it seems to have no effect. So I tried adding the `%Z` spec from `chrono`, like this:

```
{% if version %}\
    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d %H:%M:%S %Z") }}
{% else %}\
    ## [unreleased]
{% endif %}\
```

But this results in the following error:

```zsh
$ git-cliff --tag v8.1.16 > CHANGELOG.md
thread 'main' panicked at 'a Display implementation returned an error unexpectedly: Error', /private/tmp/rust-20220304-25491-7e24v/rustc-1.59.0-src/library/alloc/src/string.rs:2399:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

So I added a variable called `utc` to which I assign the `timestamp` and format it to an ISO 8601 date, and then apply the `timezone` to it, like this:

```
{% if version %}\
    {% set utc = timestamp | date(format="%Y-%m-%dT%H:%M:%S%.3fZ") %}\
    ## [{{ version | trim_start_matches(pat="v") }}] - {{ utc | date(format="%Y-%m-%d", timezone="America/Tijuana") }}
{% else %}\
    ## [unreleased]
{% endif %}\
```

The same [date from the first post](https://github.com/ignition-api/7.9/blob/main/CHANGELOG.md#7919---2022-03-16) is rendered properly using the `utc` variable approach.

```
## [7.9.19] - 2022-03-15 18:35 PDT
```

Compared to not using `utc`:

```
## [7.9.19] - 2022-03-16 01:35
```

Considering all of this, I'll reopen the issue to further discuss.

**Comment by orhun** [comment]
Thanks to your `utc` variable example, I can now see the problem here: Tera only supports ISO 8601 date strings for specifying a timezone. (It was also stated in the documentation and I somehow missed it 😮‍💨 )

So I created this PR: https://github.com/Keats/tera/pull/714

It will basically make the following example work without issues:

```
{{ timestamp | date(timezone="America/Tijuana", format="%Y-%m-%d") }}
```

> Since `timestamp` is "in seconds, from epoch", would it be possible to add a field to the `Release` struct with ISO 8601 format?

I think my PR solves this in a cleaner way. Let's hope that it will get merged soon! 🐻 

> Or can my approach be simplified?

Sure:

```
{{ timestamp | date(format="%Y-%m-%dT%H:%M:%S%.3fZ") | date(timezone="America/Tijuana") }}
```

**Comment by thecesrom** [comment]
Hello, @orhun.

Now that https://github.com/Keats/tera/pull/714 has been merged, and it appears to be included in [v1.16.0](https://github.com/Keats/tera/compare/v1.15.0...v1.16.0), when would you consider leveraging this new feature?

Thanks!

**Comment by orhun** [comment]
Hey @thecesrom,

I updated `tera` in be9e15873b470948c0afdfa3bb209ab30784c17f. Thanks for the reminder!

I'm planning to push a new release this weekend or sooner.
